atmo/celery.py

```diff
@@ -11,13 +11,38 @@
 configurations.setup()
 
```

Comment:
It would be great to have the implementation of the backoff in a single commit. At the moment it's introduced in the first commit and fixed in this one. I'll leave it to you to decide whether it's worth it or not